app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -1373,13 +1376,18 @@ def rbac_role_add_parent(node)
   # Set role record variables to new values
   def rbac_role_set_record_vars(role)
     role.name = @edit[:new][:name]
-    role.settings ||= {}
+    role.settings ||= {:restrictions => {}}
```

Comment:
not sure why I keep coming back to this line
think this is pedantic, but it has me nervous

```suggestion
    role.settings ||= {}
    role.settings[:restrictions] ||= {}
```